docs/source/cloudlabtutorial/index.rst

```diff
@@ -0,0 +1,129 @@
+
+- Go to "manage SSH keys"
+
+- Press "Load from file" and select the public key file
+
+- Once it's loaded in, select "add key"
+
+.. image:: ../images/cloudlab-ssh.png
+   :width: 600
+
+Starting a ONVM Profile With 1 Node on CloudLab
+-----------------
+
+- Click on experiments in the upper-left corner, then start an experiment
+
+- For "select a profile" you can choose the desired profile (onvm is the most updated/recommended)
+
+    If you don't have access to the GW profiles, you can find it `here <https://www.cloudlab.us/p/GWCloudLab/onvm>`_
+
+- For "parameterize" use 1 host, and for node type, you can keep it as is or select a different one. 
+
+    If you're running into trouble running the manager, selecting the c220g5 node may assist you
+    
+- For "finalize" you can just click next
+
+- For "schedule," you don't have to make a schedule, leaving it blank will start it immediately 
+
+Connecting to the Node via VSCode
+-----------------
+
+Before connecting, you must have uploaded your SSH key, and started an experiment
+You also must have these VSCode extensions:
+    Remote - SSH
+    Remote - SSH: Editing Configuration Files (may come preinstalled with Remote SSH)
+    Remote - Containers
+    Remote - WSL
+
+There are 2 easy ways to connect:
+
+1) Via a terminal
+
+- Open a VSCode terminal and cd inside your .ssh folder 
+
+- :code:`ssh -p 22 -i <privatekeyname> <user>@<address>`
+
+- Your <user>@<address> can be found by going to your experiment and clicking on "list view," it should be under "SSH Command"
+
+.. image:: ../images/ssh-connect.png
+   :width: 600
+
+2) Via a Remote Window
+
+- Open the "Remote Explorer" via the sidebar (on the left by default)
+
+- In the drop-down window at the top, select SSH Targets
+
+.. image:: ../images/vscode-remote-explorer.png
+   :width: 600
+
+- To the right of the SSH Targets bar, click the plus button, and enter :code:`ssh <user>@<address>`
+
+- Select a configuration file (recommanded to use the one in the .ssh folder as mentioned earlier)
+
+- Modify the config file so that it has the correct settings:
+    It should have :code:`Port 22` :code:`IdentityFile <privatekeyname>` and :code:`AddKeysToAgent Yes` (on seperate lines)
+    
+    You can also rename :code:`Host` to whatever you want, but :code:`HostName` must not be changed
+
+.. image:: ../images/config.png
+   :width: 600
+    
+-  If it asks you to choose an operating system, select Linux
+
+Running the ONVM Manager and Speed Tester NF on the node
+-----------------
+Once you are properly connected to the node, it's time to run the manager
+
+- First, cd into /local/onvm/openNetVM/scripts and run :code:`source setup_cloudlab.sh`
+
+- Depending on which node you're using, it will ask you to bind certain network devices to dpdk 
+    In general, you want to make sure that the 2 10 GbE devices are bound (the letters/numbers before listing the device can be used as identifiers)
```

Comment:
For this guide you only had them request a server with one node, so there isn't much point binding the NIC ports (since the user won't have another node to send traffic to this one).  I think you can have these instructions just say that for this guide we won't be working with real network traffic so we do not need to bind any ports.